### PR TITLE
drawer menu: don't show default tap color on chrome

### PR DESCRIPTION
**Pull request by sebastianbenz**
Also: remove invalid css statement from first line & reformatting

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@rowan-m can you take a look?

File: templates/css/styles.css [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /* reset */
-  everflow-wrap: break-word;
```
Comment (rowan-m) [inline templates/css/styles.css]: Is the intent to remove this or add the correct property back in?

Comment (sebastianbenz) [inline templates/css/styles.css]: the correct property is still there...this just needs to be removed

**Comment by rowan-m** [comment]
LGTM

**Comment by sebastianbenz** [comment]
Cheers!

**Pull request merged**